src/Compilers/Test/Core/Microsoft.CodeAnalysis.Test.Utilities.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFrameworks>net5.0;netcoreapp3.1;netstandard2.0;net472</TargetFrameworks>
```

Comment:
If we're removing `netstandard2.0` here we should clean up `RuntimeUtilities.cs` to remove the `#elif NETSTANDARD2_0` regions.